pkg/network/latency/summary.go

```diff
@@ -0,0 +1,41 @@
+package latency
+
+import "fmt"
+
+type distribution struct {
+	Average int64
+}
+
+type summary struct {
+	All     *distribution
```

Comment:
what does "All" mean here?

Reply:
All means all the connect results(including the successful ones and the failure ones)